README.md

```
@@ -10,7 +10,7 @@
 [![Open in Codespaces](https://img.shields.io/badge/-Open%20in%20Codespaces-black?logo=github)](https://codespaces.new/Ryan-Millard/Img2Num)
 
 <a href="https://skillicons.dev">
-  <img height="25px" src="https://skillicons.dev/icons?i=cpp,c,js" />
+  <img height="25px" src="https://skillicons.dev/icons?i=cpp,c,js,py" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)